website/blog/2023-04-08-databend-weekly-88.md

```diff
@@ -0,0 +1,184 @@
+    image_url: https://github.com/psiace.png
+---
+
+[Databend](https://github.com/datafuselabs/databend) is a modern cloud data warehouse, serving your massive-scale analytics needs at low cost and complexity. Open source alternative to Snowflake. Also available in the cloud: <https://app.databend.com> .
+
+## What's On In Databend
+
+Stay connected with the latest news about Databend.
+
+### Support Eager Aggregation
+
+Eager aggregation is a technique that can help improve the performance of queries that involve grouping and joining data. It works by partially pushing a groupby past a join, which reduces the number of input rows to the join and may result in a better overall plan.
+
+Databend recently added support for Eager aggregation. Here is an example of how it works.
+
+```
+Input:
+                expression
+                    |
+         aggregate: SUM(x), SUM(y)
+                    |
+                   join
+                    | \
+                    | (y)
+                    |
+                   (x)
+(1) Eager Groupby-Count:
+                expression
+                    |
+     final aggregate: SUM(eager SUM(x)), SUM(y * cnt)
+                    |
+                    join
+                    | \
+                    | (y)
+                    |
+                    eager group-by: eager SUM(x), eager count: cnt
+(2) Eager Split:
+                expression
+                    |
+     final aggregate: SUM(eager SUM(x) * cnt2), SUM(eager SUM(y) * cnt1)
+                    |
+                    join
+                    | \
+                    | eager group-by: eager SUM(y), eager count: cnt2
+                    |
+                    eager group-by: eager SUM(x), eager count: cnt1
+```
+
+If you are interested in learning more, please check out the resources listed below.
+
+- [PR #10716 | feat(planner): support eager aggregation](https://github.com/datafuselabs/databend/pull/10716)
+- [Paper | Eager Aggregation and Lazy Aggregation](https://www.vldb.org/conf/1995/P345.PDF)
+
+### Support All TPC-DS Queries
+
+Databend now supports all TPC-DS queries!
+
+TPC-DS is a decision support benchmark that models several generally applicable aspects of a decision support system, including queries and data maintenance. The benchmark provides a representative evaluation of performance as a general-purpose decision support system.
+
+If you are interested in learning more, please check out the resources listed below.
+
+- [PR | feat: support cte in dphyp](https://github.com/datafuselabs/databend/pull/10673)
+- [PR | chore(test): add tpcds to test](https://github.com/datafuselabs/databend/pull/10673)
+
+## Code Corner
+
+Discover some fascinating code snippets or projects that showcase our work or learning journey.
+
+### databend-driver - A driver for Databend in Rust
+
+The Databend community has developed a driver for Databend in Rust. It allows developers to easily connect to Databend and execute SQL queries using Rust.
```

Comment:
The Databend community has crafted a Rust driver that allows developers to  connect to Databend and execute SQL queries in Rust.